syft/workers/base.py

```diff
@@ -492,6 +492,7 @@ def execute_computation_action(self, action: ComputationAction) -> PointerTensor
             if type(_self) == str and _self == "self":
                 _self = self
             if sy.framework.is_inplace_method(op_name):
+                # TODO [sharan]: Can this be used to fix the inplace operations breakages??
```

Comment:
If there's work left to be done, better to create an issue than a TODO.

Reply:
Removed (it was addressed)